### PR TITLE
remove admin_enabled

### PR DESCRIPTION
**Pull request by somesylvie**
## Description
Fortify wants us to not have `admin_enabled` set to `true` on the container registry. This PR changes that setting and turns on `container_registry_use_managed_identity` instead. To test this, we deployed to both a Flexion environment (this PR env) and a CDC one (dev) and confirmed the app deployed successfully and was running

## Issue
https://github.com/CDCgov/trusted-intermediary/issues/1209

### CODE REVIEW

**Review by pluckyswan — COMMENTED** [review]
Would it be beneficial to include why Fortify wants this?

File: operations/template/app.tf [hunk 1]
```diff
@@ -53,6 +52,8 @@ resource "azurerm_linux_web_app" "sftp" {
     health_check_path                 = "/health"
     health_check_eviction_time_in_min = 5
 
+    container_registry_use_managed_identity = true
```
Comment (halprin) [inline operations/template/app.tf:55]: Down below on what is currently line `84` and `85`, we have...
```
DOCKER_REGISTRY_SERVER_USERNAME = azurerm_container_registry.registry.admin_username
DOCKER_REGISTRY_SERVER_PASSWORD = azurerm_container_registry.registry.admin_password
```

Can we remove those lines and this continues to work?

**Review by halprin — COMMENTED** [review]
Just a question on whether we can remove a small bit of code.

File: operations/template/app.tf (hunk 1 above)
Comment (somesylvie) [reply operations/template/app.tf:55]: We can try it and find out 😅

**Comment by somesylvie** [comment]
> Would it be beneficial to include why Fortify wants this?

The Fortify rec is actually pretty vague - this is all it says 
image

**Review by pluckyswan — APPROVED** [review]

**Pull request merged**